scripts/lint_jjb.py

```diff
@@ -155,6 +168,134 @@ def check_retention(job, in_file):
     return return_value
 
 
+# Ensure all jobs have a valid timed triggers entry
+# Structure:
+#   job:
+#     triggers:
+#      - timed: "@daily"
+def check_timed_trigger_value(trigger):
+
+    # If the trigger provided is a NoneType,
+    # then the job has no value, so pass it.
+    if type(trigger) == 'NoneType':
+        return_value = 1
+    # If the trigger provided is not a string,
+    # then it is not valid.
+    if type(trigger) != 'str':
```

Comment:
If the above is a `NoneType`, then it won't be a `str`, meaning it will match this also.

Reply:
Good catch - this should be elif.